src/igvfd/tests/test_upgrade_curated_set.py

```diff
@@ -16,3 +16,12 @@ def test_curated_set_upgrade_2_3(upgrader, curated_set_v2):
     value = upgrader.upgrade('curated_set', curated_set_v2, current_version='2', target_version='3')
     assert value['accession'] == 'IGVFDS0000ZZZA'
     assert value['schema_version'] == '3'
+
+
+def test_curated_set_upgrade_3_4(upgrader, curated_set_v3):
+    value = upgrader.upgrade(
+        'curated_set', curated_set_v3,
+        current_version='3', target_version='4')
+    assert value['schema_version'] == '4'
+    assert 'publication_identifiers' in value
```

Comment:
please compare the content of publication_identifiers and refrences in value, it is not sufficient to check for presence if the content was lost